requirements-core.txt

```diff
@@ -6,7 +6,7 @@ lxml==5.2.0
 matplotlib>=3.3.1
 networkx>=2.6
 nibabel>=3.2.1
-numpy==1.26.4
+numpy<=1.23.5,>=1.23.1
```

Comment:
Is there any reason for having this range?

Reply:
@wonjuleee just a mistake. i've updated.